recipe/meta.yaml

```diff
@@ -33,7 +33,7 @@ requirements:
     - llvm-openmp  # [osx]
   host:
     - python
-    # temporary pin pip to avoid missing oldest-supported-numpy error 
+    # temporary pin pip to avoid missing oldest-supported-numpy error
     - pip ==22.0.4
     - setuptools <60.0
     - cython >=0.29.24
```

Comment:
Will the change of the `cython` minimum version ( https://github.com/scikit-learn/scikit-learn/pull/25342 ) be part of the next release?

Reply:
It will be part of 1.3 as a development requirement: for bug-fix releases, we exclude contributions which might unknowingly change behaviors, which is the case of updating Cython.

Reply:
Ah ok. Yeah that makes sense. Thanks for the clarification! 🙏

Reply:
You're welcome ! I appreciate your involvement, @jakirkham. 🙂